.ci/test-spelling-unknown-words.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# This CI acceptance test is based on:
+# https://github.com/jsoref/spelling/blob/04648bdc63723e5cdf5cbeaff2225a462807abc8
+# It is conceptually `f` which runs `w` (spelling-unknown-word-splitter)
+# plus `fchurn` which uses `dn` mostly rolled together.
+set -e
+temp=`pwd`/.ci-temp
+if [ ! -d $temp ]; then
+  git clone https://github.com/jsoref/checkstyle-contribution.git $temp
```

Comment:
Please clone to .ci-temp/contribution folder. `.ci-temp` will be reused by others CI too.